src/utils/eval_tuner.ts

```diff
@@ -0,0 +1,224 @@
+import { getRandomItem } from '.'
+import { setRandomSeed } from 'bun:jsc'
+import type { Tensor } from '@shumai/shumai'
+
+setRandomSeed(Math.random() * 100000)
```

Comment:
what's the reason for this?

Reply:
Seeding the seed used by Math.random(), by default it's not seeded (other Js runtimes don't expose the ability to seed this), but Bun does.